stagiaires/florent/exercices/Logique/Exo 6.7 - somme des suivants

```diff
@@ -0,0 +1,21 @@
+Début programme
+    Variables locales
+        Tab : tab[10]
+        i :N <- 0
+        j :N <- 0
+        somme :N <- 0
+        TAILLE :N <- 10
+    Fin Variables locales
+
+    TANT QUE i <= TAILLE
+        i <- j
+        TANT QUE j <= TAILLE - i
+            somme <- somme + Tab[j]
+            j <- j + 1
+        FINTANTQUE
```

Comment:
je pense qu'il y a un problème quelque part: tu travailles avec un tableau qui commence à l'index 0 (car i est à 0) alors que je pensais qu'on devait travailler avec des tableaux qui commencent à l'index 1. A la première itération tu as un problème quand tu arrives à la dernière case qui doit être tab[9]. Elle se fait, puis j est augmenté de 1. Il vérifie la condition "j <= TAILLE - i" (car i = 0) =====> "10 <= 10 - 0" et tu te retrouves en dehors du tableau avec Tab[10] qui n'existe pas